File: chatwoot/csharp/src/OSEG.ChatwootExamples/DeleteACustomFilterExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class DeleteACustomFilterExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["userApiKey"] = "USER_API_KEY"};
        // config.ApiKey = new Dictionary<string, string> {["agentBotApiKey"] = "AGENT_BOT_API_KEY"};
        // config.ApiKey = new Dictionary<string, string> {["platformAppApiKey"] = "PLATFORM_APP_API_KEY"};

        try
        {
            new CustomFiltersApi(config).DeleteACustomFilter(
                accountId: null,
                customFilterId: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling CustomFiltersApi#DeleteACustomFilter: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/DeleteACustomFilterExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
  // apiKey: "AGENT_BOT_API_KEY",
  // apiKey: "PLATFORM_APP_API_KEY",
});

new api.CustomFiltersApi(configuration).deleteACustomFilter(
  0, // accountId
  0, // customFilterId
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling CustomFiltersApi#deleteACustomFilter:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
